static/public/javascripts/vendor/omniture.js

```diff
@@ -43,7 +43,7 @@ function s_doPlugins(s) {
 //    s.prop62 = "Mdot Guardian H.25.3 v1 20130122";
 
     /* URL */
-    s.prop61 = "D=g";
+    s.prop61=s.eVar9="D=g" // passes the page URL into both prop61 and eVar9
```

Comment:
Actually Grant this wont work as we need the primary captured value going
into the eVar not the prop. So if you can turn it round so its

s.eVar9 = "D=g";s.prop61 = s.eVar9;


Thanks


On 10 April 2015 at 15:25, Matt Kopka <matt.kopka@theguardian.com> wrote:

> Thats cools for me.  Gets the same outcome and if helps with clarity I'm
> all for that.
>
> Thanks
>
> On 10 April 2015 at 15:23, Grant Klopper <notifications@github.com> wrote:
>
>> In static/public/javascripts/vendor/omniture.js
>> <https://github.com/guardian/frontend/pull/8800#discussion_r28148078>:
>>
>> > @@ -43,7 +43,7 @@ function s_doPlugins(s) {
>> >  //    s.prop62 = "Mdot Guardian H.25.3 v1 20130122";
>> >
>> >      /* URL */
>> > -    s.prop61 = "D=g";
>> > +    s.prop61=s.eVar9="D=g" // passes the page URL into both prop61 and eVar9
>>
>> Ah, I see.
>>
>> For clarity I would prefer...
>>
>> s.prop61 = "D=g";
>> s.eVar9 = s.prop61;
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/guardian/frontend/pull/8800/files#r28148078>.
>>
>
>
>
> --
> Thanks - Matt
>
>
> (L)  0203 35 (33044)
> (M) 07508 589 888
>



-- 
Thanks - Matt


(L)  0203 35 (33044)
(M) 07508 589 888

-- 

------------------------------
Visit theguardian.com. On your mobile and tablet, download the Guardian 
iPhone and Android apps theguardian.com/guardianapp and our tablet editions 
theguardian.com/editions.  Save up to 57% by subscribing to the Guardian 
and Observer - choose the papers you want and get full digital access. 
 Visit subscribe.theguardian.com

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396